torchtune/models/phi3/_sentencepiece.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         self.pad_id = self.special_tokens["<|endoftext|>"]
 
         # During generation, stop when eos_id is encountered
-        self.stop_tokens = {self.eos_id}
+        self.stop_tokens = [self.eos_id]
```

Comment:
Curious: why?

Reply:
PyTorch has trouble converting set to a tensor via the `torch.tensor` API